### ISSUE TITLE
Changing `sbt.global.settings` and `sbt.global.plugins` does not work with Coursier

### ISSUE THREAD

**Issue by jvican** [coursier/coursier#773]
When I change the `sbt.global.plugins`, coursier does not correctly resolve my build. This happens because coursier assumes that `sbt.global.dir` is the base directory of the global plugins directory, when this is not true (see options `-Dsbt.global.plugins` et al in `sbt.BuildPaths`).
  
This happens in `Tasks.scala`:
```
    // FIXME get the 0.13 automatically?
    val defaultRawPattern = s"$${sbt.global.base.uri-$${user.home.uri}/.sbt/$sbtVersion}/plugins/target" +
      "/resolution-cache/" +
      "[organization]/[module](/scala_[scalaVersion])(/sbt_[sbtVersion])/[revision]/resolved.xml.[ext]"

    // seems to be required in more recent versions of sbt (since 0.13.16?)
    val extraRawPattern = s"$${sbt.global.base.uri-$${user.home.uri}/.sbt/$sbtVersion}/plugins/target" +
      "(/scala-[scalaVersion])(/sbt-[sbtVersion])" +
      "/resolution-cache/" +
      "[organization]/[module](/scala_[scalaVersion])(/sbt_[sbtVersion])/[revision]/resolved.xml.[ext]"
```
  
And this is the error I get:
  
```
coursier.ResolutionException: Encountered 1 error(s) in dependency resolution:
    org.scala-sbt:global-plugins;sbtVersion=0.13;scalaVersion=2.10:0.0:
        not found:
            /root/.ivy2/local/org.scala-sbt/global-plugins/scala_2.10/sbt_0.13/0.0/ivys/ivy.xml
            https://repo1.maven.org/maven2/org/scala-sbt/global-plugins_2.10_0.13/0.0/global-plugins-0.0.pom
            https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/global-plugins/scala_2.10/sbt_0.13/0.0/ivys/ivy.xml
            https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/org.scala-sbt/global-plugins/scala_2.10/sbt_0.13/0.0/ivys/ivy.xml
[error] (*:coursierResolutions) coursier.ResolutionException: Encountered 1 error(s) in dependency resolution:
[error]     org.scala-sbt:global-plugins;sbtVersion=0.13;scalaVersion=2.10:0.0:
[error]         not found:
[error]             /root/.ivy2/local/org.scala-sbt/global-plugins/scala_2.10/sbt_0.13/0.0/ivys/ivy.xml
[error]             https://repo1.maven.org/maven2/org/scala-sbt/global-plugins_2.10_0.13/0.0/global-plugins-0.0.pom
[error]             https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/global-plugins/scala_2.10/sbt_0.13/0.0/ivys/ivy.xml
[error]             https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/org.scala-sbt/global-plugins/scala_2.10/sbt_0.13/0.0/ivys/ivy.xml
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? [error] Filed to generate bloop config with sbt 0.13.
[error] (frontend / integrationSetUpBloop) Filed to generate bloop config with sbt 0.13.
[error] Total time: 179 s, completed Feb 9, 2018 12:01:23 PM 
```
  
Changing the global plugins dir has lots of value, especifically when source dependencies are used.

**Comment by alexarchambault** [comment]
I agree that this can be changed, PR welcome!

**Comment by alexarchambault** [comment]
This should work better since https://github.com/coursier/sbt-coursier/pull/35 and https://github.com/coursier/sbt-coursier/pull/47 (included in sbt-coursier >= `1.1.0-M13-2`), which make sbt-coursier not rely on these patterns any more. Re-open if that's still a problem.
